nuxeo-runtime/nuxeo-runtime-metrics/src/main/java/org/nuxeo/runtime/metrics/MetricsDescriptor.java

```diff
@@ -140,16 +160,22 @@ public String prefix() {
         }
 
         protected boolean filter(String name) {
-            if (filters == null || filters.isEmpty()) {
-                String filters = Framework.getProperty(FILTER_PROPERTY, DEFAULT_METRICS);
-                return matchesFilter(name, Arrays.stream(filters.split(",")));
+            if ((whitelistMetrics == null || whitelistMetrics.isEmpty())
+                    && (blacklistMetrics == null || blacklistMetrics.isEmpty())) {
+
+                Stream<String> reportedMetrics = Arrays.stream(
+                        Framework.getProperty(WHITELIST_FILTER_PROPERTY, DEFAULT_WHITELIST_METRICS).split(","));
+                Stream<String> ignoredMetrics = Arrays.stream(
+                        Framework.getProperty(BLACKLIST_FILTER_PROPERTY, DEFAULT_BLACKLIST_METRICS).split(","));
+                return matchesFilter(name, reportedMetrics, ignoredMetrics);
```

Comment:
Just think about that, we don't cache to `whitelistMetrics` nor `blacklistMetrics` result of `nuxeo.conf` properties retrieval. Since it used to be the default case, maybe we should take care of it. wdyt?